### PR TITLE
SIMS #21 Student contact information

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
Added the contact information to the student profile and SIN number in preparation for #68.

The bellows validation frameworks were considered and the one on this PR is the VueValidate.

- https://vee-validate.logaretm.com/v4/
- https://vuelidate.js.org/

Veutify supports both frameworks as 3rd party validators, as shown here: https://vuetifyjs.com/en/components/forms/#vee-validate, but we may not need a 3rd party validator when we change to Veutify, as shown in the below image (copied from the above link).

image

Both seem to be well accepted in the community:
https://www.reddit.com/r/vuejs/comments/emcsoa/veevalidate_vs_vuelidate/

vee-validate has more contributors and more starts on GitHub.

The "Save" button is just displaying an alert with the data. This should be enough to unblock the #78 that @fwpushan is currently working on.
The SIN specific validation will be added together with #68.
The "address line 2" is not required.
I am thinking of doing a POC using also the vee-validate so the team can have a better idea and define the best one to continue to use alongside #68.

Please find below how the screen looks like now.
image

image

### CODE REVIEW

**Review by abschwenker — COMMENTED** [review]
lookin good

File: sources/packages/web/src/components/ValidatedInput.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <div :class="{ 'validation-error': property.$invalid }">
+      <slot></slot>
```
Comment (fwpushan) [inline sources/packages/web/src/components/ValidatedInput.vue:4]: I thing slot is deprecated and v-slot is introduced. 
https://vuejs.org/v2/guide/components-slots.html#Deprecated-Syntax
Can you check

**Review by fwpushan — COMMENTED** [review]
LGTM, only one concern.

File: sources/packages/web/src/components/ValidatedInput.vue (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/web/src/components/ValidatedInput.vue:4]: Yes, that makes sense. I will change it.

Comment (andrewsignori-aot) [reply sources/packages/web/src/components/ValidatedInput.vue:4]: Hi @fwpushan, the page that you mentioned is related to Vue 2 where it states the new way to do it in Vue 2 in comparison with the previous version.
The slots on Vue 3 are to be used it is in the code right now.
https://v3.vuejs.org/guide/component-slots.html#slot-content

**Review by 2011sandeepraj — APPROVED** [review]
LGTM

**Pull request merged**